### ISSUE TITLE
[tool_crash] FileSystemException: Cannot open file, OS Error: Operation not supported, errno = 45

### ISSUE THREAD

**Issue by axeldeveloper** [flutter/flutter#79833]
## Command
```
flutter build ios-framework --xcframework --no-universal --output=/Dev/flutter/innovea-pdt-008-flutter-2
```

## Steps to Reproduce
1. ...
2. ...
3. ...

## Logs
FileSystemException: Cannot open file, OS Error: Operation not supported, errno = 45
```
#0      _File.throwIfError (dart:io/file_impl.dart:635:7)
#1      _File.openSync (dart:io/file_impl.dart:479:5)
#2      _File.writeAsBytesSync (dart:io/file_impl.dart:604:31)
#3      ForwardingFile.writeAsBytesSync (package:file/src/forwarding/forwarding_file.dart:124:16)
#4      FileSystemUtils.copyDirectorySync (package:flutter_tools/src/base/file_system.dart:81:17)
#5      FileSystemUtils.copyDirectorySync (package:flutter_tools/src/base/file_system.dart:84:9)
#6      FileSystemUtils.copyDirectorySync (package:flutter_tools/src/base/file_system.dart:84:9)
#7      BuildIOSFrameworkCommand._produceFlutterFramework (package:flutter_tools/src/commands/build_ios_framework.dart:324:23)
#8      BuildIOSFrameworkCommand.runCommand (package:flutter_tools/src/commands/build_ios_framework.dart:194:15)
<asynchronous suspension>
#9      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:1157:12)
<asynchronous suspension>
#10     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1009:27)
<asynchronous suspension>
#11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
<asynchronous suspension>
#12     AppContext.run (package:flutter_tools/src/base/context.dart:149:12)
<asynchronous suspension>
#13     CommandRunner.runCommand (package:args/command_runner.dart:197:13)
<asynchronous suspension>
#14     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:278:9)
<asynchronous suspension>
#15     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
<asynchronous suspension>
#16     AppContext.run (package:flutter_tools/src/base/context.dart:149:12)
```
```
[✓] Flutter (Channel stable, 2.0.4, on Mac OS X 10.15.6 19G73 darwin-x64, locale pt-BR)
    • Flutter version 2.0.4 at /Users/Axel/development/flutter
    • Framework revision b1395592de (4 days ago), 2021-04-01 14:25:01 -0700
    • Engine revision 2dce47073a
    • Dart version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/Axel/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[!] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods installed but not working.
        You appear to have CocoaPods installed but it is not working.
        This can happen if the version of Ruby that CocoaPods was installed with is different from the one being used to
        invoke it.
        This can usually be fixed by re-installing CocoaPods.
      To re-install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google
    Chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.55.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension can be installed from:
      🔨 https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[!] Connected device
    ! No devices available

! Doctor found issues in 3 categories.

```

## Flutter Application Metadata
**Type**: app
**Version**: 1.0.0+1
**Material**: true
**Android X**: false
**Module**: false
**Plugin**: false
**Android package**: null
**iOS bundle identifier**: null
**Creation channel**: stable
**Creation framework version**: 2783f8e2e14efec8b7e08f668dde61c40d128c24
### Plugins
flutter_plugin_android_lifecycle-1.0.11
flutter_sms-2.1.1
flutter_user_agent-1.2.2
image_picker-0.6.7+14
path_provider-1.6.24
path_provider_linux-0.0.1+2
path_provider_macos-0.0.4+6
path_provider_windows-0.0.4+3
share-0.6.5+4
shared_preferences-0.5.12+4
shared_preferences_linux-0.0.2+4
shared_preferences_macos-0.0.1+11
shared_preferences_web-0.1.2+7
shared_preferences_windows-0.0.1+3
url_launcher-5.7.10
url_launcher_linux-0.0.1+4
url_launcher_macos-0.0.1+9
url_launcher_web-0.1.5+1
url_launcher_windows-0.0.1+3
video_player-0.10.12+5
video_player_web-0.1.4+1
wakelock-0.1.4+2

**Comment by pedromassangocode** [comment]
I found similar error message in https://github.com/flutter/flutter/issues/31111.

**Comment by axeldeveloper** [comment]
"in many cases the downgrade or migrate does not work"

I solved the problem by downloading version 1.24 from the Dev channel

export PATH = "$ PATH: / Users / axel / development / flutter_1_24 / bin"

flutter clean
flutter run
